frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -68,6 +68,7 @@ export const TARGET_AREA_TO_NAME = {
     toolbar: 'Toolbar & Heatmaps',
     surveys: 'Surveys',
     web_analytics: 'Web Analytics',
+    mobile: 'Mobile',
```

Comment:
I didn't add a separate view in case people didn't remember to check both inboxes.

I did [update our routing rule](https://posthoghelp.zendesk.com/admin/objects-rules/rules/triggers/14505447085723) to include Mobile tickets to the Replay inbox

Reply:
I thought that the view was the equivalent of the routing rule, all good then.